README.md

```diff
@@ -58,29 +64,45 @@ The following tutorial applies to Linux / MacOS development environments. Visit
 
 type: "cd .." to go back to the toynet-flask directory
 
+
+populate environment variables for testing
+
+```
+ source environment/env-dev
 ```
-# populate environment variables for testing
-$ source environment/env-dev
 
-# Populate environment variables for production
-$ source environment/env-prod
+Populate environment variables for production
+
+```
+ source environment/env-prod
+```
 
-# create reclass_network
-$ docker network create reclass_network
+create reclass_network
+
+```
+ docker network create reclass_network
+```
+
+ creates instance/toynet.sqlite
+
+```
+ flask init-db 
+```
 
-# creates instance/toynet.sqlite
-$ flask init-db 
 Initialized the database.
 
-$ flask run
+```
+ flask run
+```
+
  * Serving Flask app "flasksrc" (lazy loading)
  * Environment: development
  * Debug mode: on
  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
  * Restarting with stat
  * Debugger is active!
  * Debugger PIN: 220-725-712
- ```
+
```

Comment:
It's pretty common to keep this kind of output in a code block, but I see how it can be confusing since we don't have output for other commands. Let's just delete it so things are consistent (and do the same for `initialized the database` above).

Reply:
You will then need to build the image. The default way, which will work with the Makefile here, is to go into the `toynet_mininet` submodule and run:
```
$ make test-image
```

These instructions are not clear. Is the test-image the image your building to work with the Makefile or is it the Makefile.

Reply:
Yeah, including it as a separate block looks good.